src/main/java/com/jcloisterzone/board/EdgePattern.java

```diff
@@ -143,14 +231,26 @@ public boolean isBridgeAllowed(Location bridge) {
         return true;
     }
 
+    /**
+     * Returns the appropriate {@link EdgeType} to replace the one currently on the given {@code side}.
+     *
+     * @param side the side of interest
+     * @return the appropriate {@link EdgeType} to replace the one currently on the given {@code side}
+     */
     private EdgeType getBridgeReplacement(Location side) {
         switch (at(side)) {
-        case FARM: return EdgeType.ROAD;
-        case UNKNOWN: return EdgeType.UNKNOWN;
-        default: throw new IllegalArgumentException();
+            case FARM: return EdgeType.ROAD;
```

Comment:
sorry, I missed this one

please don't make extra indent for case statement
see https://stackoverflow.com/questions/4509039/why-the-strange-indentation-on-switch-statements

also same level is default style for Eclispe IDE (and probably also others)

Reply:
Fixed it where you mentioned and also in Rotation.java